src/amplitude/amplitude.ts

```diff
@@ -0,0 +1,37 @@
+import { AmplitudeEvent } from './taxonomy-events';
+
+declare global {
+    /* tslint:disable:no-unused-variable */
+    interface Window {
+        veilarbpersonflatefsAmplitude: ({
+            origin,
+            eventName,
+            eventData
+        }?: {
+            origin: string;
+            eventName: string;
+            eventData?: Record<string, unknown>;
+        }) => Promise<void>;
+    }
+    /* tslint:enable:no-unused-variable */
+}
+
+export const maskereFodselsnummer = (data?: Record<string, unknown>) => {
+    const maskertObjekt = JSON.stringify(data)
+        .replace(/"\d{11}"/g, '"***********"')
```

Comment:
Er det meningen at vi skal kjøre samme `replace`-funksjonen to ganger?

Reply:
Oppdaget det med testen faktisk, at man må plukke opp både string og number eventuelt. Følte det uansett var lettere å gjøre det sånn enn å si at man skal gå igjennom alle keys som igjen kan ha verdi objekt så må man dypere og dypere :P

Men altså for å forklare helt konkret må den kunne fange opp begge disse variantene: '{"fnrString":"12345678910","fnrNumber":12345678910}'